API/Queries/Query.cs

```diff
@@ -0,0 +1,30 @@
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Backend.API.Data;
+using Backend.API.Data.GeographicalModels;
+using Backend.API.Services;
+using GraphQL;
+using GraphQL.Types;
+
+namespace Backend.API.Queries
+{
+    public class Query : ObjectGraphType
+    {
+        public Query(IDataRetrievalService dataRetrievalService)
+        {
+            Field<TodoType>("todo", arguments: new QueryArguments()
+            {
+                new QueryArgument<IntGraphType>
+                    {Name = "id", Description = "The ID of the todo to retrieve.", DefaultValue = 1}
+            }, resolve: context => dataRetrievalService.GetTodo(context.GetArgument<int>("id")));
+
+            Field<ListGraphType<TodoType>>("todos", resolve: _ => dataRetrievalService.GetTodos());
+            Field<ListGraphType<MetApiCompactType>>("geomet", resolve: _ =>
+            {
+                var jsonAsString = File.ReadAllText(Tools.Tools.AssetsPath + "MetApiTestData.json");
+                return JsonSerializer.Deserialize<MetApiCompact.Root[]>(jsonAsString);
```

Comment:
Could this be handled in the `dataRetrievalService`, or eventually in a new mocked data retrieval service which can be enabled by a feature flag or similar?

Reply:
Yes indeed. I think the easiest way to go forward with this is setting an environment variable, which is documented [here](https://blog.jetbrains.com/dotnet/2018/11/08/using-net-core-launchsettings-json-rundebug-apps-rider/). This way we can have mocked or real data while in development mode. The ConfigureServices method in Startup.cs will then use the appropriate service depending on the env-variable.

We could also add simple logic so that when the application is in production, it only uses real data.

Reply:
That should definitely be doable! Good suggestion.